File: pickupAndDeliveryBack/src/main/java/com/Bestanome/Livraisons/Livraison.java
package com.Bestanome.Livraisons;

public class Livraison {
  private Long pickup;
  private Long destination;
  private int dureeEnlevement;
  private int dureeLivraison;

  public Livraison(Long idStart, Long idEnd, int dEnlevement, int dLivraison){
    this.pickup = idStart;
    this.destination = idEnd;
    this.dureeEnlevement = dEnlevement;
    this.dureeLivraison = dLivraison;
  }

  public Long[] itineraire(){
    return new Long[] {this.pickup, this.destination};
  }
}

File: pickup-and-delivery-front/src/types/Livraison.ts
export interface LivraisonsEntrepot {
  livraisons: Livraison[];
  idEntrepot: number
}

export interface Livraison {
  id: number;
  pickup: number;
  destination: number;
  dureeEnlevement: number;
  dureeLivraison: number;
}